NightscoutServiceKit/RemoteCommands/RemoteCommandValidation.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  RemoteCommandValidation.swift
+//  LoopKit
+//
+//  Created by Bill Gestrich on 2/25/23.
+//  Copyright © 2023 LoopKit Authors. All rights reserved.
+//
+
```

Comment:
A remote command may have multiple validators (ex: OTP, expiration, ...) And what we validate may vary by the command type.